azure-pipelines.perf-test.yml

```diff
@@ -154,3 +158,52 @@ jobs:
           uniqueId: 'perfComment9423'
 
       - template: .devops/templates/cleanup.yml
+
+  - job: PerfTestReactComponents
+    displayName: Perf test for @fluentui/react-components
+    dependsOn: CheckIfPackagesAffected
+    condition: eq(dependencies.CheckIfPackagesAffected.outputs['PackagesAffected.ReactComponentsPackageAffected'], true)
+    workspace:
+      clean: all
+    steps:
+      - template: .devops/templates/tools.yml
+      - task: Bash@3
+        inputs:
+          filePath: yarn-ci.sh
+        displayName: yarn
+
+      # Build v8 Perf Test
+      - script: |
+          yarn lage build --to perf-test-react-components --verbose
+        condition: eq(variables.isPR, true)
+        workingDirectory: apps/perf-test-react-components
```

Comment:
curious why workingDirectory needs to be set ? `lage` can be invoked from root

Reply:
This is true, I was just copying from initial pipeline implementation - will remove these!